CHANGELOG.md

```diff
@@ -4,16 +4,17 @@
 
 ### Added
 - Add optional custom_prompt arg to the Text2CypherRetriever class.
-  
+
+### Changed
+- `GraphRAG.search` method first parameter has been renamed `query_text` (was `query`) for consistency with the retrievers interface.
+- Made `GraphRAG.search` method backwards compatible with the query parameter, raising warnings to encourage using query_text instead.
```

Comment:
Not sure we need a line in changelog for that, maybe rephrasing the previous one would be nicer. Something like:

```
`query` parameter in GraphRAG.search` was deprecated and will be removed in the next major release. Use `query_text` instead. 
```